de-de/c-de.html.markdown

```diff
@@ -0,0 +1,880 @@
+---
+language: c
+filename: learnc-de.c
+contributors:
+    - ["caminsha", "https://github.com/caminsha"]
+lang: de-de
+---
+
+Ach, C. Immer noch **die** Sprache für modernes High-Performance Computing.
+
+C ist wahrscheinlich die niedrigste Programmiersprache, welche die meisten
+Programmierer je brauchen werden. Die Geschwindigkeit von C ist enorm, allerdings
+muss man sich stets der maneullen Speicherverwaltung bewusst sein.
```

Comment:
manuellen

Reply:
Didn't see this typo even though I read the whole article about 3 times. Thanks!